models/view/src/index.ts

```diff
@@ -782,6 +782,20 @@ export function createModel (builder: Builder): void {
     view.action.ShowPreview
   )
 
+  createAction(
+    builder,
+    {
+      action: view.actionImpl.ExportToPdf,
+      label: view.string.ExportToPdf,
+      // icon: view.icon.Delete,
+      category: view.category.General,
+      input: 'any',
+      target: core.class.Doc,
+      context: { mode: ['context', 'browser'], group: 'edit' }
```

Comment:
`group` should not be `edit`. It affects the position of the action in the context menu.